scripts/write-configs.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const jsYaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+const tsNode = require('ts-node');
+
+tsNode.register();
+
+const kangarooConfig = require(path.join(process.cwd(), 'kangaroo.config.ts')).default;
+
+if (!process.env.KANGAROO_DEV) {
+  // CHECK THAT NO DEFAULT VALUES ANYMORE
+  if (kangarooConfig.appId === 'org.holochain.kangaroo-electron')
```

Comment:
```suggestion
  if (kangarooConfig.appId.includes('org.holochain.kangaroo-electron'))
```

Reply:
Well, that's gonna make it a bit more annoying to find a different name during testing it locally because I can't just append something arbitrary :sweat_smile: . But maybe worth it?

Reply:
No this was just to reflect that the product name included `.configtest`. All good.